src/rviz/default_plugin/marker_utils.cpp

```diff
@@ -76,4 +84,289 @@ MarkerBase* createMarker(int marker_type, MarkerDisplay* owner, DisplayContext*
   }
 }
 
+bool checkMarkerMsg(const visualization_msgs::Marker& marker, MarkerDisplay* owner)
+{
+  std::stringstream ss;
```

Comment:
Move this below the fast-track return in line 92.
Allocating memory for `stringstream` is costly and should only done when needed.

Reply:
Done in eaa5419